### DISCUSSION TITLE
How to set logLevel to control debugging verbosity?

### DISCUSSION THREAD

**Discussion by cardboardcode** [STORM-IRIT/OpenGR#94] [Tutorials]
Hi there, 

May i enquire what is the right way to silence the verbosity?

After checking through the OpenGR API, the verbosity seems to be linked to the enum variable in Utils: `logLevel`. Here is the [link](https://storm-irit.github.io/OpenGR/a00102.html#ad36376e2e9f9c71561dca43867be64d4).

Is there a way to programmatically set the `logLevel` from **Verbose** to **NoLog** when running the pose alignment process?

**Comment by nmellado** [comment]
Hi,

You can set the `loglevel` when creating the `Logger`:

```c++
constexpr Utils::LogLevel loglvl = Utils::Verbose;
Utils::Logger logger(loglvl);
/*...*/
score = computeAlignment<MatcherType> (options, logger, set1, set2, mat, sampler, visitor);
```
Example: https://github.com/STORM-IRIT/OpenGR/blob/master/demos/Super4PCS/super4pcs_test.cc#L129
